### PR TITLE
Editor: Use GDScript as the default language in ScriptCreateDialog.

### PR DESCRIPTION
**Pull request by 27thLiz**

### CODE REVIEW

**Comment by karroffel** [comment]
I feel discriminated

**Review by akien-mga — APPROVED** [review]

File: editor/script_create_dialog.cpp [hunk 1]
```diff
@@ -359,20 +359,26 @@ ScriptCreateDialog::ScriptCreateDialog() {
...
+		for (int i = 0; i < language_menu->get_item_count(); i++) {
 			if (language_menu->get_item_text(i) == last_selected_language) {
 				language_menu->select(i);
 				break;
-			} else
-				language_menu->select(0);
+			}
+		}
+	} else
+		language_menu->select(default_lang);
```
Comment (bojidar-bg) [inline editor/script_create_dialog.cpp]: Mind if you brace it? I think it would look better....

Comment (akien-mga) [inline editor/script_create_dialog.cpp]: Agreed. I wonder if we should enforce braces for single-line if/else in clang-format?

Comment (Hinsbart) [inline editor/script_create_dialog.cpp]: Fixed

**Pull request merged**